hooks.mk

```diff
@@ -42,4 +42,4 @@ deploy:
 integration-tests:
 	@echo ".: 🩺 Starting integration test in kind namespace 'integration-tests'."
 	kubectl -n integration-tests delete scans --all
-	cd ../../tests/integration/ && npm ci &&	npx --yes --package jest@$(JEST_VERSION) jest --verbose --ci --colors --coverage --passWithNoTests ${hook-prefix}s/$(name).test.js
+	cd .. && npm ci && cd $(name)/${hook-prefix} && npm run test --package jest@$(JEST_VERSION)
```

Comment:
Is there a system behind the naming convention, if a var is lower-kebab-case or upper-snake-case? Usually in Makefiles is upper-snale-case like in Shell scripts. I recommend one style, unless there is a really good and documented reason for this exception.